cypress/e2e/tests/pages/global-settings/branding.spec.ts

```diff
@@ -369,6 +369,18 @@ describe('Branding', { testIsolation: 'off' }, () => {
       expect(background).to.satisfy((b) => b.startsWith(settings.primaryColor.newRGB));
     });
 
+    // check that login page has new styles applied
+    // https://github.com/rancher/dashboard/issues/10788
+    loginPage.goTo();
+
+    loginPage.submitButton().self().should('have.css', 'background').should((background: string) => {
+      expect(background).to.satisfy((b) => b.startsWith(settings.primaryColor.newRGB));
+    });
+
+    cy.login();
```

Comment:
I think it would be good to reload a page and check that the styles are still the same just in case?

Reply:
done